build.gradle

```diff
@@ -25,7 +25,6 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
 	implementation 'org.flywaydb:flyway-core'
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
-	implementation 'org.springframework.boot:spring-boot-starter-data-rest'
```

Comment:
Removed as we do not want to expose REST endpoints from the repositories themselves.  This is useful sometimes if you do not want to put a controller in front of the repositories but for our use case we should always be creating:

- `Controller` - Layer responsible for handling the incoming HTTP request
- `Service` - Layer responsible for performing business logic
- `Repository` - Layer responsible for interfacing with the DB (inserts, deletes, queries etc...)